components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -367,7 +364,8 @@ func certValidator(cm *corev1.ConfigMap, dataKey string, log logr.Logger) error
 	odhCertData, ok := cm.Data[dataKey]
 	if !ok || odhCertData == "" {
 		// Print a warning if odh-ca-bundle.crt data is empty
-		return fmt.Errorf("Warning: %s data is empty", dataKey)
+		log.Info(fmt.Sprintf("%s data are empty", dataKey))
+		return fmt.Errorf("%s data are empty", dataKey)
```

Comment:
Just asking here - so effectively this will lead into the error message in the log as was before (just on line 328), correct? And it is because we actually cannot mount the data as a file with an empty value, is that true?

Reply:
Added a bool variable in case of empty data, to handle it. Now in case of empty data we fall here https://github.com/atheo89/kubeflow/blob/RHOAIENG-4165/components/odh-notebook-controller/controllers/notebook_webhook.go#L322